packages/eslint-plugin/tests/rules/space-infix-ops.test.ts

```diff
@@ -0,0 +1,98 @@
+/* eslint-disable @typescript-eslint/internal/plugin-test-formatting */
```

Comment:
Prefer something like this:

https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/tests/rules/brace-style.test.ts#L1-L4

```suggestion
/* eslint-disable eslint-comments/no-use */
// this rule tests spacing, which prettier will want to fix and break the tests
/* eslint "@typescript-eslint/internal/plugin-test-formatting": ["error", { formatWithPrettier: false }] */
/* eslint-enable eslint-comments/no-use */
```